qpmodel/optimizer.cs

```diff
@@ -303,7 +303,6 @@ public class CMemoGroup
         //
         public Dictionary<PhysicProperty, (CGroupMember member, double cost)> minMember_ { get; set; }
             = new Dictionary<PhysicProperty, (CGroupMember, double)>();
-        public double minIncCost_ { get; set; }
```

Comment:
make minIncCostNullProperty_ a geet only property.

Reply:
class NullProperty: PhysicProperty{} so when we enforce property, it is never null.